bpf/lib/dbg.h

```diff
@@ -119,6 +119,7 @@ enum {
 	DBG_SK_ASSIGN,		/* arg1: result
 				 * arg2: unuseds
 				 */
+	DBG_L7_LB,
```

Comment:
Please add the args here in comments so readers can interpret the fields and double-check correctness.

Reply:
```
-	DBG_L7_LB,
+	DBG_L7_LB,		/* arg1: saddr (last 4 bytes for IPv6)
+				 * arg2: daddr (last 4 bytes for IPv6)
+				 * arg3: proxy port (in host byte order)
+				 */
```
Not sure if using the same enum value for IPv4 and IPv6 works out?